### ISSUE TITLE
Path not being populated

### ISSUE THREAD

**Issue by jesper-bylund** [metalsmith/collections#63]
Not sure if this is related to the partial build bug, or CLI. But I'm building via the CLI interface and no posts in collection are receiving paths:

```
{
  "source": "./views/blog-src",
  "destination": "./views/blog-pub",
  "plugins": {
    "metalsmith-collections": {
      "posts": {
        "pattern": "./views/blog-src/posts/*.html",
        "sortBy": "date",
        "reverse": "true"
      },
      "latestPosts": {
        "pattern": "./views/blog-src/posts/*.html",
        "sortBy": "date",
        "reverse": "true",
        "limit": "10"
      }
    },
    "metalsmith-permalinks": {
      "pattern": ":date/:title"
    },
    "metalsmith-markdown": true,
    "metalsmith-layouts": {
      "engine": "ejs",
      "default": "post.ejs",
      "directory": "views",
      "version": "lkebgw46346lbj",
      "user": false,
      "static": true,
      "userID": false
    }
  }
}
```

This is what I get:

```
{ title: 'test post',
  date: Thu Jan 01 2015 01:00:00 GMT+0100 (CET),
  publishDate: Thu Jan 01 2015 01:00:00 GMT+0100 (CET),
  modifyDate: Thu Jan 01 2015 01:00:00 GMT+0100 (CET),
  author: 'Jesper Bylund',
  collection: [ 'posts' ],
  contents: '<p>This is just a test of the watch thing!</p>\n',
  mode: '0644',
  stats: 
   { dev: 16777220,
     mode: 33188,
     nlink: 1,
     uid: 501,
     gid: 20,
     rdev: 0,
     blksize: 4096,
     ino: 30144996,
     size: 169,
     blocks: 8,
     atime: Sun Jul 03 2016 15:35:52 GMT+0200 (CEST),
     mtime: Sun Jul 03 2016 15:28:18 GMT+0200 (CEST),
     ctime: Sun Jul 03 2016 15:28:18 GMT+0200 (CEST),
     birthtime: Sun Jul 03 2016 11:26:01 GMT+0200 (CEST) } }
```

**Comment by samrocksc** [comment]
@raelmiu do your actual paths have a layout attached to them? It looks like there's no layout and/or template to show them, so they don't have a path.

**Comment by varya** [comment]
I'm having the sam issue. Any news on it?

**Comment by ericlathrop** [comment]
Looks like `path` was removed in https://github.com/segmentio/metalsmith-collections/pull/62 and you need to use something like https://www.npmjs.com/package/metalsmith-paths to replace it.

**Comment by leviwheatcroft** [comment]
#62 hasn't actually been merged into master.

The current npm version doesn't set `path` - master branch includes a commit to populate `path` property which has not been pushed in an npm version.

So you could use `npm install segmentio/metalsmith-collections` to grab the current master, but of course that might break in 10 minutes if #62 is pulled in. 

I just made PR #66 which will allow compatibility with `metalsmith-paths` while preserving the current functionality.
